src/action/base/create_user.rs

```diff
@@ -328,18 +328,28 @@ async fn create_user_macos(name: &str, uid: u32, gid: u32) -> Result<(), ActionE
         ".",
         "-create",
         &format!("/Users/{name}"),
-        "IsHidden",
-        "1",
+        "RealName",
+        name,
     ])
     .await?;
     execute_dscl_retry_on_specific_errors(&[
         ".",
         "-create",
         &format!("/Users/{name}"),
-        "RealName",
-        name,
+        "IsHidden",
+        "1",
     ])
-    .await?;
+    .await
+    .or_else(|e| {
+        if let ActionErrorKind::CommandOutput { ref output, .. } = e {
+            if output.status.signal() == Some(9) {
+                tracing::warn!("Failed to automatically mark the user as hidden. See: https://dtr.mn/mark-user-hidden");
```

Comment:
This has the potential for a lot of spam if it happens on every user (up to, say, 32 times? :P)

Could we maybe use an `AtomicBool` so that it only gets printed once, or somehow collect the users it failed on into a vec and at the end of the user creation say `tracing::warn!(?failed_users, "failed to mark these users as hidden blah blah")`?

Reply:
Oh, getting fancy.

Reply:
Good idea. Done!